client/layout/guided-tours/config-elements/step.js

```diff
@@ -117,6 +121,11 @@ export default class Step extends Component {
 			this.setStepPosition( nextProps, shouldScrollTo );
 			this.watchTarget();
 		} );
+		if ( ! nextProps.keepRepositioning ) {
+			clearInterval( this.repositionInterval );
```

Comment:
If the purpose of current fix is to account for elements repositioning while loading, maybe we could clear the interval after fixed number of attempts?
Also, I don't see when we are going to hit this case with current setup (the `keepRepositioning` prop will always be `true`).